plugins/DataLoadRlog/rlog_parser.hpp

```diff
@@ -2,16 +2,27 @@
 
 #include <PlotJuggler/messageparser_base.h>
 #include <capnp/dynamic.h>
+#include <capnp/schema.h>
+#include <capnp/serialize.h>
 #include <iostream>
 
+#ifndef DYNAMIC_CAPNP
+#define DYNAMIC_CAPNP  // Do not depend on generated log.capnp.h structure
+#endif
+#include "../../3rdparty/opendbc/can/common.h"
```

Comment:
Is this relative path needed? Didn't you add the include dir in the cmake config?